lib/TestReportHandler.rb

```diff
@@ -127,11 +127,27 @@ def pass
     return 
   end 
 
+# Might need to alter ‘score’ calculation based on including/not including setup or teardown failures
+# Or, add the option to decide whether to include them
+
   def finalize
     report.status = 'completed'
     report.score = ((1 - @num_failures / @num_tests.to_f) * 100).round(2)
     report.result = (report.score == 100 ? 'pass' : 'fail')
     report.issued = DateTime.now
+    
+    num_passes = 0
+    report.test.each do |test| 
+          test.action.each do |ac| 
+               [your code here]
```

Comment:
```suggestion
              num_passes += 1 if ac.operation&.result&.code == 'pass'
              num_passes += 1 if ac.assert.result.code == 'pass' if ac.assert
```
I intended the `[your code here]` to indicate that score counting you wrote should be placed there. My error, I didn't make that clear enough. However, did you pull down the changes to your local branch and run the driver? This should've raised some error via the driver.

Reply:
I understand the meaning of `[your code here]` but got lost when I pull it into my VS Code. My bad. I will commit the suggestion here, then try to add my code into it.

Reply:
I understand `[your code here]` but got lost when pulled it in VS Code. Looks like your another suggestion below would take the lines 142 and 143 out. I will commit them all, then add the code and try to commit and push into PR.